server/go/api_apis_service_impl.go

```diff
@@ -95,7 +110,13 @@ func (s *ApisApiService) GetPostProcessedOpenApiSpec(ctx context.Context, namesp
 	analytics.SendAnonymousInfo(ctx, s.kuskClient.K8sClient(), "GetPostProcessedOpenApiSpec")
 	api, err := s.kuskClient.GetApi(namespace, name)
 	if err != nil {
-		return Response(http.StatusInternalServerError, err), err
+		switch err {
```

Comment:
within the server package, this switch can be extracted into its own function that returns a `Response` to prevent duplicating the switch everywhere

Reply:
I concur